CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ It is based on Tidepool Blip 1.27.
 
 ### Fixed
 - YLP-2951 Patient data not accessible (infinite loading)
+- YLP-2967 Broken wizard (bolus + meal) display in the Daily view
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** Version to update? (Decision TBC)‏

Reply:
Seen with Yasmina: OK for 3.3.0